purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -54,6 +58,15 @@ data class PaywallColor(
             null
         },
     )
+
+    constructor(@ColorInt colorInt: Int) : this(
+        stringRepresentation = String.format(Locale.US, COLOR_HEX_FORMAT, COLOR_WHITE_HEX and colorInt),
+        underlyingColor = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            Color.valueOf(colorInt)
+        } else {
+            null
```

Comment:
Can this be removed with https://github.com/RevenueCat/purchases-android/pull/1269/commits/1b922768e39b62e9c7fbad157d038d70f35d29be?